client/src/livingdocs-component-app/app.js

```diff
@@ -46,6 +50,36 @@ export class App {
     if (this.selectedItem) {
       await this.loadPreview();
     }
+
+    // load any additional stylesheets defined for themeing or font-face loading as @font-face doesn't work within ShadowRoot (used for the preview)
+    try {
```

Comment:
Imo the stylesheets should be loaded before the preview is loaded (or at least at the same time), otherwise there could be a flash of font change. maybe extract the stylesheet loading in a separate function and call it at the beginning of `attached` without await, so the stylesheets get loaded parallel to the preview?